docs/modules/icamauth/icamauth.md

```diff
@@ -0,0 +1,218 @@
+# icamauth module
```

Comment:
Shouldn't this file be named `README.md` so it shows up by default when you're in the directory?

Reply:
right !